phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixLoggingMetricsIT.java

```diff
@@ -114,7 +114,7 @@ public void testPhoenixMetricsLoggedOnCommit() throws Exception {
         loggedConn.commit();
         assertTrue("Mutation write metrics for not found for " + tableName2,
                 mutationWriteMetricsMap.get(tableName2).size() > 0);
-        assertMutationMetrics(tableName2, NUM_ROWS, mutationWriteMetricsMap);
+        assertMutationMetrics(tableName2, NUM_ROWS, true, mutationWriteMetricsMap);
```

Comment:
where did you make the API changes? i didn't find the extra boolean param

https://github.com/apache/phoenix/blob/2a2d9964d29c2e47667114dbc3ca43c0e264a221/phoenix-core/src/it/java/org/apache/phoenix/monitoring/BasePhoenixMetricsIT.java#L106

Reply:
Forgot to check-in the file,Added now.